bin/bb/cmd/testdata/expected/test_migrate_after

```diff
@@ -0,0 +1,38 @@
+--
```

Comment:
We likely could have a better naming convention to organize the test cases.

Thinking of we will add more test cases in the future, how we make each test case more identifiable and manageable.